setParameters.m

```diff
@@ -32,30 +32,38 @@
     % MRI settings
     cfg = setMRI(cfg);
 
-    cfg.audio.channels = 2;
-
     %% Experiment Design
-    cfg.names              = {'static', 'motion'};
-    cfg.possibleDirections = [-1 1]; % 1 motion , -1 static
-    cfg.numBlocks          = size(cfg.possibleDirections, 2);
-    cfg.numRepetitions     = 1;      % AT THE MOMENT IT IS NOT SET IN THE MAIN SCRIPT
+
+    cfg.design.names              = {'static', 'motion'};
+    cfg.design.possibleDirections = [-1 1]; % 1 motion , -1 static
+    cfg.design.numBlocks          = size(cfg.possibleDirections, 2); % TO CHECK
+    cfg.design.numRepetitions     = 1;  % AT THE MOMENT IT IS NOT SET IN THE MAIN SCRIPT
+    cfg.design.nbEventsPerBlock   = 12;
 
     %% Timing
-    cfg.IBI                = 0; % 8;
-    cfg.ISI                = 0.1;    % Time between events in secs
-    cfg.onsetDelay         = 1;      % Number of seconds before the motion stimuli are presented
-    cfg.endDelay           = 1;      % Number of seconds after the end all the stimuli before ending the run
+
+    % FOR 7T: if you want to create localizers on the fly, the following must be
+    % multiples of the scanneryour sequence TR
+    %
+    % IBI
+    % block length = (cfg.eventDuration + cfg.ISI) * cfg.design.nbEventsPerBlock
+
+    % Time between blocs in secs
+    cfg.timing.IBI = 1.8 * 3; % 8;
+    % Time between events in secs
+    cfg.timing.ISI = 0.1;
+    % Number of seconds before the motion stimuli are presented
+    cfg.timing.onsetDelay = .1;
+    % Number of seconds after the end all the stimuli before ending the run
+    cfg.timing.endDelay = .1;
 
     %% Auditory Stimulation
 
-    % expParameters.experimentType    = 'Gratings';  %Dots/Gratings % Visual modality is in RDKs %NOT USED IN THE MAIN SCIPT
-    cfg.speedEvent        = 8;       % speed in visual angles
-    cfg.numEventsPerBlock = 12;      % Number of events per block (should not be changed)
-    cfg.eventDuration     = 10;
+    cfg.audio.channels = 2;
 
-    %% Task
+    %% Task(s)
 
-    cfg.task.name = 'visual localizer';
+    cfg.task.name = 'auditory localizer';
 
     % Instruction
     cfg.task.instruction = '1-Detect the RED fixation cross\n \n\n';
```

Comment:
Do we want to change the instructions to "detect longer sounds" or something like that?

Reply:
yes, let me open an issue and will change this when I'll implement it